packages/terra-notification-dialog/src/NotificationDialog.module.scss

```diff
@@ -42,98 +33,120 @@
     outline: none;
     width: 100%; // Needed for IE to wrap content correctly in flexbox
   }
-
-  .header-body {
-    background-color: var(--terra-notification-dialog-header-background-color, #f9f9f9);
-    border-bottom: var(--terra-notification-dialog-header-border-bottom, 0.0625rem solid #dedfe0);
-    font-size: var(--terra-notification-dialog-header-font-size, 1.125rem);
-    padding: var(--terra-notification-dialog-header-padding, 0.8rem);
-    text-align: left;
+  
+  .floating-header-background {
+    height: var(--terra-notification-dialog-floating-background-height, 5.78571rem);
+    position: absolute;
+    width: 100%;
+    z-index: -1;
+  }
+  
+  .alert {
+    background-image: var(--terra-notification-dialog-alert-background-image, linear-gradient(to bottom, #ffdad9, #fff));
+    border-top: var(--terra-notification-dialog-alert-border-top, 7px solid #ea2f2f);
   }
 
-  .footer-body {
-    background-color: var(--terra-notification-dialog-footer-background-color, #f9f9f9);
-    border-top: var(--terra-notification-dialog-footer-border-top, 0.0625rem solid #dedfe0);
-    flex: 0 0 auto;
-    font-size: var(--terra-notification-dialog-footer-font-size, 0.8rem);
-    padding: var(--terra-notification-dialog-footer-padding, 0.28571rem 0.35714rem);
+  .error {
+    background-image: var(--terra-notification-dialog-error-background-image, linear-gradient(to bottom, #e4e2e2, #fff));
+    border-top: var(--terra-notification-dialog-error-border-top, 7px solid #c8cacb);
```

Comment:
Border-top is not getting applied for error variant.

![image](https://user-images.githubusercontent.com/45097016/82213485-d4271600-9931-11ea-91de-31a7d99c6bdd.png)

Reply:
The dark grey color at the top of the dialog is the top-border that is being applied. When you remove border-top the error dialog appears like this:
![Screen Shot 2020-05-18 at 8 24 25 AM](https://user-images.githubusercontent.com/14099737/82218187-1ffd9f00-98e1-11ea-9c13-7df24efac1cb.png)

Reply:
i was seeing a different color in callouts so i had this question.

![image](https://user-images.githubusercontent.com/45097016/82226917-bb742b80-9944-11ea-89d5-6ac4e12e5a4a.png)

Reply:
Here are the callouts I am using for open-source theme: https://app.zeplin.io/project/59b1bfeae515657fa8268300/screen/5a5f7e7b4f9f24b874e0f3ad

Which callout are you referencing?

Reply:
Ok, i think i was referencing the old one.